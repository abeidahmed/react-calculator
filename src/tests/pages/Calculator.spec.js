import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../../pages/Calculator';

describe('Calculator', () => {
  test('renders home page', () => {
    const component = renderer.create(<Calculator />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
