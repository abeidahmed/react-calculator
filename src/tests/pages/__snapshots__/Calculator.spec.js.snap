// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders home page 1`] = `
<div
  className="sc-eCImvq etHGKp"
>
  <div
    className="sc-bdvvaa epAHfG"
  >
    <p
      className="sc-gsDJrp gIEJOc"
    >
      _
    </p>
    <p
      className="sc-gsDJrp iVfYxk"
      fontSize={42}
    >
      0
    </p>
  </div>
  <div>
    <div
      className="sc-hKwCoD bHqEiN"
    >
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="sc-dkPtyc hwJofF"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div
      className="sc-hKwCoD bHqEiN"
    >
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="sc-dkPtyc hwJofF"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="sc-hKwCoD bHqEiN"
    >
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="sc-dkPtyc hwJofF"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="sc-hKwCoD bHqEiN"
    >
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="sc-dkPtyc hwJofF"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="sc-hKwCoD bHqEiN"
    >
      <button
        className="sc-dkPtyc intobT"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="sc-dkPtyc izUORN"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="sc-dkPtyc hwJofF"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
  <span
    style={
      Object {
        "display": "none",
      }
    }
  />
</div>
`;
